examples/reference/panes/Plotly.ipynb

```diff
@@ -226,7 +238,69 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Plotly Layouts\n",
+    "## Streaming\n",
+    "\n",
+    "You can stream updates by replacing the entire Figure object. To stream efficiently though you should use patching technique described above.\n",
+    "\n",
+    "You can stream periodically using `pn.state.add_periodic_callback`."
+   ]
```

Comment:
As I don't believe I understand efficiency of dictionaries please check if the streaming example is the most efficient way to stream to plotly as users will be using this as the reference example.